src/device/mod.rs

```diff
@@ -110,50 +118,107 @@ fn test_list_function() {
     assert!(devices.len() > 0);
 }
 
-/// Enables a Network Manager device.
+/// Connects a Network Manager device.
 ///
 /// # Examples
 ///
-/// ```
-/// use network_manager::device;
-/// use network_manager::manager;
-/// let manager = manager::new();
-/// let devices = device::list(&manager).unwrap();
-/// let device = &devices[0];
-/// let state = device::enable(&manager, device, 10).unwrap();
-/// println!("{:?}", state);
-/// ```
-pub fn enable(manager: &NetworkManager, c: &Device, t: i32) -> Result<DeviceState, String> {
-    // Enable device
-
-    if t != 0 {
-        // Wait until the device state is 'Activated' or
-        // until the time has elapsed
+/// # ```
+/// # use network_manager::device;
```

Comment:
Yes, we need to figure it out. I think it is better to create a separate Github issue for it, since it is a broader problem which may involve other tests as well. (It breaks because of the doctests run in the connection module). I will create one now.